ksql-serde/src/test/java/io/confluent/ksql/serde/delimited/KsqlDelimitedSerializerTest.java

```diff
@@ -104,7 +105,7 @@ public void shouldSerializedTopLevelPrimitiveIfValueHasOneField() {
         .field("id", Schema.OPTIONAL_INT64_SCHEMA)
         .build();
 
-    final Serializer<Object> serializer = new KsqlDelimitedSerializer();
+    final Serializer<Object> serializer = createSerializer();
```

Comment:
can't this line be deleted? i.e. it will just use the `serializer` created in `setUp`.?

Same for others...